editor/src/components/canvas/ui-jsx.test-utils.tsx

```diff
@@ -93,11 +96,7 @@ const FailJestOnCanvasError = () => {
   const stableCallback = React.useCallback((newRuntimeErrors: Array<RuntimeErrorInfo>) => {
     // we have new runtime errors, let's take the tests down
     if (newRuntimeErrors.length > 0) {
-      const errorMessage = `The test run threw the following canvas error(s): ${newRuntimeErrors
-        .map((e) => e.error)
-        .join(', ')}`
-      fail(errorMessage)
-      throw new Error(errorMessage) // just in case (?) the fail() wouldn't be effective
+      expect(newRuntimeErrors[0]?.error ?? null).toEqual(null)
```

Comment:
I don't understand why this has been changed?

Reply:
riiight. this is jest 27 related, not tsconfig :/

Reply:
What I mean is that previously this would explicitly fail the test with the error message, whereas now you're checking if the error is null

Reply:
I don't remember quite well, but I _think_ this printed a way nicer and more concise output, something about failing with `fail` was less nice, but my memory is fuzzy about it